src/commands/create.js

```diff
@@ -193,6 +194,25 @@ export default asyncCommand({
 			spinner.succeed('Done!\n');
 		}
 
+		if (argv.less || argv.sass || argv.stylus) {
+			const { less, sass, stylus } = argv;
+			const srcDir = path.resolve(target, 'src');
+
+			let extension;
+
+			if (less) extension = '.less';
+			if (sass) extension = '.scss';
+			if (stylus) extension = '.styl';
+
+			await copy(srcDir, srcDir, {
```

Comment:
we can use `renamer` though but it is recommended to use it globally. In that case, I'm not really sure if this is a good idea or not.